parsers/IN_MH.py

```diff
@@ -219,9 +219,9 @@ def fetch_production(zone_key='IN-MH', session=None, target_datetime = None,
     # string together all image sections and recognize resulting line
     imgs_line = np.hstack( list(np.asarray(i) for i in imgs[:]) )  
     imgs_line = Image.fromarray( imgs_line)
-    text = pytesseract.image_to_string(imgs_line, lang='digits_comma', config='--psm 7')
+    digit_reader = pytessy.PyTessy(language='digits_comma')
```

Comment:
On my linux system I get a `NotImplementedError('PyTessy: Library search on Linux is not implemented yet.')`
To avoid this you have to provide pytessy with the exact paths. For me `digit_reader = pytessy.PyTessy(lib_path='/usr/bin/tesseract', data_path='/usr/share/tesseract-ocr/4.00/tessdata', language='digits_comma')` works. 
@Kongkille  how would this work for the backend?
For more information on the pytessy class: https://github.com/hyperrixel/pytessy/blob/master/source/pytessy.py

Reply:
Sorry for the late reply @GitMatze! The backend is on Linux, so we'll need to specify the explicit library paths, I'm not entirely sure what we would do, but if you could get it working on your machine, I'm sure we can tweak it to the backend?

Reply:
Is there anything I can do to help this?

Reply:
@FelixDQ would you know which changes need to be made for it to work on our backend?

Reply:
It's running using the [python:3.6 docker image](https://hub.docker.com/_/python). We have a list of setup commands that we run when building the container. If you can provide the command(s), that we should add, then it would be perfect.

Otherwise, if you have the steps needed to install the library / find the path on your own Linux system, then we can probably get it working on the server.

Reply:
You only need to adjust the paths passed to the pytessy.Pytessy() object in order for it to access the tesseract library. Depending on where tesseract is installed on the backend the paths should look pretty similar..

Reply:
Hi @GitMatze ,
In our linux container, the binary `/usr/bin/tesseract` exists but `find_library` in https://github.com/hyperrixel/pytessy/blob/master/source/pytessy.py#L154 doesn't return anything.
Reading https://docs.python.org/3/library/ctypes.html, and running from our python instance in our linux container, I get:

```bash
>>> find_library('tesseract')
'libtesseract.so.4'
```
but `find_library('/usr/bin/tesseract')` returns None
Note that libtesseract.so.4 is located in /usr/lib/x86_64-linux-gnu/ (using the path didn't help either.
Passing just `libtesseract` as `lib_path` doesn't work, due to this check: https://github.com/hyperrixel/pytessy/blob/master/source/pytessy.py#L265-L269

Is this supposed to work on Linux? Seems like `find_library` returns the full path on MacOS, but not on Linux.

Reply:
@GitMatze are you able to help here?

Reply:
I've updated the pyproject setup, so now we also get the error on CI.

> NotImplementedError: PyTessy: Library search on Linux is not implemented yet.